packages/errors/src/codes.ts

```diff
@@ -267,8 +268,8 @@ export const SOLANA_ERROR__INVARIANT_VIOLATION_SWITCH_MUST_BE_EXHAUSTIVE = 99000
  * is not safe, for a variety of reasons covered here: https://stackoverflow.com/a/28818850
  */
 export type SolanaErrorCode =
-    | typeof SOLANA_ERROR__TRANSACTION_MISSING_SIGNATURES
-    | typeof SOLANA_ERROR__TRANSACTION_SIGNATURE_NOT_COMPUTABLE
+    | typeof SOLANA_ERROR__TRANSACTION_SIGNATURES_MISSING
+    | typeof SOLANA_ERROR__TRANSACTION_FEE_PAYER_SIGNATURE_MISSING
```

Comment:
Just noticing this now, but this export list should probably be either alphabetical or in the same order as above.

Reply:
I _think_ this one does, but it also does some other annoying stuff, so yeah scoping it to here is probably best.
https://typescript-eslint.io/rules/sort-type-constituents/

Reply:
Totally fine with adding such a sorter to `@solana/eslint-config-solana`.